migration/mule/jms-message-rollback-and-redelivery/README.md

```diff
@@ -0,0 +1,67 @@
+# JMS message rollback and redelivery
+
+This example shows how to implement jms rollback and redelivery within the WSO2 Integration Studio. While when we passing the jms messages inside the WSO2 EI, following failures could be happening.
+* Failures during message mediation process.
+* Message sending failure,due to endpoint is unavailability.
+
+The JMS message passing flow in this example is configured with the ActiveMQ MOM. The requirement is if any of the above failures happen, the message should not process further and send to end point, rather it should roll back. After a specific number of unsuccessful failures happen the message should send to ActiveMQ topic.
+## Assumptions
+
+This document describes the details of the example within the context of WSO2 Integration Studio, editor to develop WSO2 EI artifacts. This document assumes that you are familiar with WSO2 EI and the [Integration Studio interface](https://ei.docs.wso2.com/en/latest/micro-integrator/develop/WSO2-Integration-Studio/). To increase your familiarity with Integration Studio, consider completing one or more [WSO2 EI Tutorials](https://ei.docs.wso2.com/en/latest/micro-integrator/use-cases/integration-use-cases/).
+
+## Example Use Case
+
+In this example, there is a JMS message consuming inbound endpoint implemented inside WSO2 EI. It listens to the ActiveMQ Queue. After consuming the JMS message by the Inbound endpoint, which rollbacks and retries to deliver the same message to the topic reside in the ActiveMQ. This rollback process is handled by `OnErrorPropagate` sequence.  
+
+The number of JMS message delivery count and filtering process is handled by sequence called `choice`. After a specific number of unsuccessful attempts to commit (4 in this case), it sends the message successfully.
+
+## Set Up and Run the Example
+
+Follow the steps in this procedure to create and run this example in your own instance of Integration Studio. You can create template applications straight out of the box in Integration Studio and tweak the configurations of the use case-based templates to create your own customized applications in WSO2 Integrator.
+
+1. Start WSO2 Integration Studio. See [Installing WSO2 Integration Studio](https://ei.docs.wso2.com/en/latest/micro-integrator/develop/installing-WSO2-Integration-Studio/) 
+2. In your menu in Studio, click the File menu. In the File menu select the **Import...** item.
+3. In the Import window select the **Existing WSO2 Projects into workspace** under **WSO2** folder.
+4. Browse and select the file path to the downloaded sample of this github project (`jms-message-rollback-and-redelivery` folder of the downloaded github repository)
+5. For setup the ActiveMQ with WSO2 EI please refer [Setting up the Micro Integrator with ActiveMQ](https://ei.docs.wso2.com/en/latest/micro-integrator/setup/brokers/configure-with-ActiveMQ/#setting-up-the-micro-integrator-with-activemq).
```

Comment:
This doc only describes how to setup `JMS Transport`, not the inbound endpoint that we have used in this sample. So linking it is wrong.